README.md

```
@@ -737,6 +747,294 @@ function rejectTransferOwners(bytes calldata _remark) external;
 
 For more information on Token Registry and Title Escrow contracts **version v5**, please visit the readme of [TradeTrust Token Registry V5](https://github.com/TradeTrust/token-registry/blob/master/README.md)
 
+#### c) Obligation Registry (BoE)
+
+> **New:** Obligation Registry supports electronic Bill of Exchange (BoE) style documents. It mirrors the classic Transferable Records / Title Escrow pattern, but uses **`TrustVCToken`** + **`ObligationEscrow`** (v5-style only — there is no Obligation v4 path).
+>
+> Prefer the high-level SDK in `@trustvc/trustvc/obligation-registry-functions` for deploy, mint, lifecycle, transfers, return-to-issuer, status reads, and document verification. Low-level Typechain factories are available from `@trustvc/trustvc/obligation-registry`.
+
+#### Lifecycle
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)